vertx-web-client/src/main/java/examples/WebClientExamples.java

```diff
@@ -761,4 +761,44 @@ public static void clientSideConsistentHashing(Vertx vertx, int servicePort) {
 
     server.listen(servicePort);
   }
+
+  public static void receiveResponseAsServerSentEvents(Vertx vertx, int servicePort) {
+    WebClient client = WebClient.create(vertx, new WebClientOptions().setDefaultPort(servicePort).setDefaultHost("localhost"));
+
+    HttpServer server = vertx.createHttpServer()
+        .requestHandler(req -> {
+          req.response().setChunked(true);
+          // set headers
+          req.response().headers().add("Content-Type", "text/event-stream;charset=UTF-8");
+          req.response().headers().add("Connection", "keep-alive");
+          req.response().headers().add("Cache-Control", "no-cache");
+          req.response().headers().add("Access-Control-Allow-Origin", "*");
+          int count = Integer.parseInt(req.getParam("count"));
+          vertx.setPeriodic(50, new Handler<Long>() {
+            private int index = 0;
+
+            @Override
+            public void handle(Long timerId) {
+              if (index < count) {
+                String event = String.format("event: event%d\ndata: data%d\nid: %d\n\n", index, index, index);
+                index++;
+                req.response().write(event);
+              } else {
+                vertx.cancelTimer(timerId);
+                req.response().end();
+              }
+            }
+          });
+        });
```

Comment:
You can remove the server setup part, I think. It makes the example quite long and what we're showing is how to use the WebClient Sse body codec

Reply:
I thought the gola was to have a 'running' example, I will do that